airbyte-workers/src/main/java/io/airbyte/workers/helper/ConnectionHelper.java

```diff
@@ -74,7 +79,21 @@ public ConnectionRead updateConnection(final ConnectionUpdate connectionUpdate)
           .withMemoryRequest(connectionUpdate.getResourceRequirements().getMemoryRequest())
           .withMemoryLimit(connectionUpdate.getResourceRequirements().getMemoryLimit()));
     } else {
-      newConnection.withResourceRequirements(workerConfigs.getResourceRequirements());
+      final io.airbyte.config.ResourceRequirements resourceRequirements = Optional.ofNullable(persistedSync.getResourceRequirements())
```

Comment:
Don't we want to leave them blank so we use the workerConfig at job creation time, not in the connection update?

Reply:
Updated